### PR TITLE
Clean up addr_full

### PR DESCRIPTION
**Pull request by Cj-Malone**
This is gonna take a while...

### CODE REVIEW

**Comment by iandees** [comment]
Thanks for working on this. Do you want me to turn off "number of spiders" check so you can see output from these?

**Comment by Cj-Malone** [comment]
Not for me. I'm checking them locally so I know all these work. If you want the stats, feel free.

File: locations/spiders/baptist_health_arkansas.py [hunk 1]
```diff
@@ -13,43 +12,31 @@ class BaptistHealthArkansasSpider(scrapy.Spider):
...
+                        "indexName": "wp_posts_location",
+                        "params": "query=&hitsPerPage=500&maxValuesPerFacet=150&page=0&facets=%5B%22city%22%2C%22facility_type%22%5D&tagFilters=",
+                    }
+                ]
+            },
             callback=self.parse_stores,
         )
 
     def parse_stores(self, response):
-        data = json.loads(json.dumps(response.json()))
```
Comment (Cj-Malone) [inline locations/spiders/baptist_health_arkansas.py:43]: I found this funny.

Comment (iandees) [reply locations/spiders/baptist_health_arkansas.py:43]: Yea, not sure why people did this.

**Pull request merged**